packages/react/src/utils/types.ts

```diff
@@ -0,0 +1,21 @@
+import type { InboxNotification } from '@novu/js';
+import type { BaseNovuUIOptions } from '@novu/js/ui';
+
+export type Notification = InboxNotification;
+export type NotificationsRenderProps = (notification: Notification) => React.ReactNode;
+
+export type InboxDefaultProps = {
```

Comment:
❓  Can we name this "DefaultInboxProps" as "Default" specified the inbox and not the props?

Reply:
Sorry, didn't understood what do you mean by Default